### DISCUSSION TITLE
docfx.console build in Visual Studio does not respect memberLayout=separatePages

### DISCUSSION THREAD

**Discussion by michfuchs** [dotnet/docfx#9225] [Q&A] [answered]
I'm using the nuget package docfx.console in VisualStudio. 

`package id="docfx.console" version="2.59.4"`

My docfx version is 2.70.3.

With this, I have to run the usual command line once to serve the site on localhost. 

`docfx docfx.json --serve`

But after that, all I need to do in order to update the _site output is build the project containing the package. This seems both faster and more convenient than always running the serve command, and has worked well. 

But this build does not seem to respect the memberLayout configuration in docfx.json:
`  "metadata": [
    {
      "memberLayout": "separatePages",
      ...
`
This will only be picked up by the command line, not the project build. 

Maybe this is a result of the version difference? As I understand the memberLayout configuration was added around 2.64, and the nuget package is older. Is the nuget package no longer released?

Any suggestions for how I can keep using the setup with the nuget package/project build?

**Comment by KalleOlaviNiemitalo** [comment, marked as answer]
`docfx.console` is indeed no longer released; see <https://github.com/dotnet/docfx/discussions/9103>.

In 2.59.4, the [memberpage](https://www.nuget.org/packages/memberpage/2.59.4) plugin would have to be used instead of `"memberLayout": "separatePages"`, but I don't know the details.

**Comment by michfuchs** [comment]
Thank you, your answer linked me up to #8605, which works perfectly for what I need to do.
